src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java

```diff
@@ -67,6 +70,17 @@ public CaseDetails save(String authorisation, Claim claim) {
         }
     }
 
+    public CaseDetails requestMoreTimeForResponse(
+        String authorisation,
+        Claim claim,
+        LocalDate newResponseDeadline
+    ) {
+        CCDCase ccdCase = this.caseMapper.to(claim);
```

Comment:
do you need to provide all the fields or can you just provide the new response deadline and the more time requested?

Reply:
I think nobody tested with just response deadline.

Reply:
Worth trying but not necessarily blocker.